docs/build/getting-started/quickstarts/testing.md

```diff
@@ -0,0 +1,102 @@
+
+# Introduction to Testing in Cadence
+
+Testing is an essential part of smart contract development to ensure the correctness and reliability of your code. In Cadence, the testing framework provides a convenient way to write tests for Cadence programs, allowing you to verify the functionality of your smart contracts.
+
+## Install Flow CLI
+
+The [Flow CLI](../../../tools/flow-cli/index.md) is the primary tool for developing, testing, and deploying smart contracts written in Cadence to the Flow network. You can install the Flow CLI by following the [installation instructions](../../../tools/flow-cli/install.md) provided by the Flow documentation.
+
+## Create a new project
+
+In your preferred code editor, create a new directory for your project and navigate to it in the terminal. Then 
+initialize a new Flow project by running the command `flow init`. This will create a `flow.json` file that contains the project configuration.
```

Comment:
Short explanation of what flow.json is and configures with a link to reference. 🙏🏼 
Additionally we might need a page with better reference of flow.json and it's available configuration

Reply:
added a reference to `tools/flow-cli/flow.json/configuration.md`